src/main/java/me/totalfreedom/totalfreedommod/admin/AdminList.java

```diff
@@ -238,6 +240,34 @@ public boolean isAdminImpostor(Player player)
         return getEntryByName(player.getName()) != null && !isAdmin(player);
     }
 
+    public boolean isCommandSpyActive(Player player)
```

Comment:
This method does to little. It is redundant and all dependent operations should probably inline any operations.